src/test/java/teammates/test/pageobjects/AdminAccountManagementPage.java

```diff
@@ -92,4 +92,9 @@ public boolean isTableVisible() {
         }
         return result;
     }
+
+    public void waitForPageToFinishLoadingOnNonDevEnvironment() {
+        By currentPageEntryCountSpan = By.id("currentPageEntryCount");
+        waitForElementPresence(currentPageEntryCountSpan, 180);
```

Comment:
Hmm ok, means if the loading ever > 1 minute, we need to do it by pagination + AJAX in the future?